content/docs/0.17.x/reference/cli/commands/keptn_configure_monitoring.md

```diff
@@ -12,7 +12,7 @@ Configures a monitoring provider
 Configure a monitoring solution for the deployments managed by Keptn.
 
 **Note:** If you are executing *keptn configure monitoring dynatrace*, the service flag is optional since Keptn automatically detects the services of a project. 
-See [Monitoring with Dynatrace](../../../../../install/monitoring/dynatrace/install/) for more information.
+See https://keptn.sh/docs/install/monitoring/ for more information.
```

Comment:
```suggestion
See [Install Dynatrace for Monitoring](https://keptn.sh/docs/install/monitoring/dynatrace/install) for more information.
```
I did a quick scan of that doc and didn't quickly see this info so we should check that sometime.